dashboard/app/assets/stylesheets/application.scss

```diff
@@ -716,6 +716,12 @@ $small-footer-standard-width: 400px;
   }
 }
 
+/* Override footer position on some pages, but not published project views */
+#page-small-footer .small-footer-base {
```

Comment:
Can you explain why this doesn't affect the published projects view?

Reply:
That view just doesn't have a `#page-small-footer` element at all. I checked to see if i could verify that ID was only used in the level show view, but it's actually from a `_page_small_footer.html.haml` that's conditionally included in `application.html.haml`, which makes it difficult to tell where else it might actually show up